esp-ieee802154/src/pib.rs

```diff
@@ -227,15 +227,29 @@ fn ieee802154_set_multipan_hal(pib: &Pib) {
     }
 }
 
-fn ieee802154_txpower_convert(txpower: i8) -> u8 {
-    const IEEE802154_TXPOWER_VALUE_MAX: i8 = 13;
-    const IEEE802154_TXPOWER_VALUE_MIN: i8 = -32;
+// https://github.com/espressif/esp-idf/blob/release/v5.3/components/hal/esp32c6/include/hal/ieee802154_ll.h
+#[cfg(all(feature = "esp32c6", not(feature = "esp32h2")))]
```

Comment:
This cfg is sus, aren't `esp32c6` and `esp32h2` mutually exclusive anyway?

Use `cfg_if` to create a branch for `esp32c6` and another for `esp32h2`.

Reply:
Good call, added a fixup and squashed in d00b474f